packages/frontend/components/ui/avatar.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cls from 'classnames';
+
+function Avatar({ type, className, image }) {
+  const avatarType = {
+    xs: 'w-8 h-8',
+    normal: 'w-11 h-11',
+    lg: 'w-14 h-14',
+  };
+
+  return (
+    <div
+      className={cls(
+        avatarType[type],
+        'overflow-hidden rounded-full',
+        className
+      )}
+    >
+      <img
+        className="bg-no-repeat bg-center bg-contain"
+        src={image}
+        alt="noopener norefferer"
+      />
```

Comment:
@omerozkilic can you add here "what if there is no image" situation? what if I haven't add image yet?

Reply:
I never thought of it before. Should I do something like a placeholder for this?

Reply:
I commit this request to avatar branch.